src/webpack-plugin/generate-keys.ts

```diff
@@ -1,4 +1,4 @@
-import { TranslocoConfig } from '@ngneat/transloco-utils';
+import { TranslocoGlobalConfig as TranslocoConfig } from '@ngneat/transloco-utils';
```

Comment:
Can you please remove the alias and use the `TranslocoGlobalConfig` directly?

Reply:
sure. will do it later, maybe 2hrs later.